jailer/src/cgroup.rs

```diff
@@ -158,7 +158,7 @@ impl Cgroup {
             }
         }
 
-        let keys_len = found_controllers.keys().len();
+        let keys_len = found_controllers.len();
```

Comment:
The name of the variable should probably be updated as well as it does not hold the keys length any more (it wasn't necessarily a good name to start with anyway).

Reply:
The name of the variable 'keys_len' has been replaced with  'found_controllers_len'.